### ISSUE TITLE
Incorrect beta subscript on slide #35

### ISSUE THREAD

**Issue by ehumph** [fhdsl/DaSEH#233]
There are two beta-sub-2 on the slide. One 2 should be a 3

**Comment by ehumph** [comment]
Note - # of visits as the outcome should probably be modeled with a poisson regression. Rate is a better choice for linear regression, since it's technically bounded between 0 and 1. Mea culpa.

**Comment by avahoffman** [comment]
Yeah might revise not to have year either. Might have to think more deeply about these!

**Comment by carriewright11** [comment]
could maybe add this resource - it's pretty good: https://tilburgsciencehub.com/topics/analyze/regression/linear-regression/regression-summary-output-r/
